pom.xml

```diff
@@ -153,8 +153,9 @@
 				<groupId>${project.groupId}</groupId>
 				<artifactId>element-connector</artifactId>
 				<version>${project.version}</version>
-				<scope>test</scope>
 				<classifier>tests</classifier>
+				<scope>test</scope>
```

Comment:
this doesn't look right to me. The managed dependency should probably cover the standard artifact which is required for compiling. FMPOV this should be:
````
<dependency>
 <groupId>${project.groupId}</groupId>
 <artifactId>element-connector</artifactId>
 <version>${project.version}</version>
</dependency>
````
If a submodule requires the *test-jar* for executing tests, it should declare this dependency explicitly (but this should not be the default).

Reply:
OK, my interpretation of dependency management is, that if several modules (but maybe not all) have a dependency to something (shared dependency), that dependency is managed in the parent's POM. Therefore junit or hamcrest are also managed in the parents POM.

The test-jars are some special cases. Usually test-stuff is not used outside. But as mentioned in #238 and #241 the element-connector contains some "test only" stuff, which is also used by test stuff outside of the element-connector. currently core and scandium use it, therefore I put it in the parent POM to manage that shared dependency.